src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/System.Net.NetworkInformation.Functional.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <StringResourcesPath>../../src/Resources/Strings.resx</StringResourcesPath>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Linux;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-FreeBSD</TargetFrameworks>
     <IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
```

Comment:
@ericstj with what we discussed, how would I express that `Browser` is not compatible when using `NetCoreAppCurrent`? In such cases I think I would still need to mimic the the src's configurations and remove Browser.

Reply:
This thing wasn't compiling for Browser before, I would expect that condition to work the same way.

Reply:
Why do you think this project wasn't compiling for Browser before locally?

Reply:
Because TargetFrameworks was `$(NetCoreAppCurrent)` not `$(NetCoreAppCurrent)-Browser` so this would build during the browser vertical, but compile *for* `$(NetCoreAppCurrent)` not `$(NetCoreAppCurrent)-Browser`.  I'm missing what changes here would require us to target `$(NetCoreAppCurrent)-Browser`

Reply:
What I'm trying to achieve here is to avoid the unnecessary Browser build completely and remove the `IgnoreForCI` proeprty but I can't express that with `$(NetCoreAppCurrent)` as the TFM as that is compatible with Browser.

Reply:
For that you would need to have a TFM for all supported OSs and exclude the platformless TFM and the Browser TFM.

Reply:
jep, from above:

> In such cases I think I would still need to mimic the the src's configurations and remove Browser.

Reply:
Can you use a placeholder TargetFramework? `$(NetCoreAppCurrent);_$(NetCoreAppCurrent)-Browser`

Reply:
No, we removed the concept and implementation of placeholders in the configuration system as it caused issues during restore and because it wasn't necessary anymore with ProjectReferences in OOBs.

Reply:
I see, then an exclusion?
https://github.com/dotnet/runtime/blob/40b8e3de457e8f31fc2754e24640b4bb51486ca7/src/libraries/tests.proj#L109

IMHO we should try to minimize the way we disable tests.  My preference would be some indication in the source, so that we get reporting over the number of tests that are disabled.

Reply:
> I see, then an exclusion?

We use project exclusions for temporary issues only... Similar to ActiveIssue in code. This doesn't solve exactly what I was asking for, which is a mechanism to not compile the Browser configuration at all which would still happen when you invoke the project directly.